Library/Homebrew/keg.rb

```diff
@@ -399,6 +399,10 @@ def delete_pyc_files!
     find { |pn| pn.delete if pn.extname == ".pyc" }
   end
 
+  def delete_pyo_files!
+    find { |pn| pn.delete if pn.extname == ".pyo" }
+  end
```

Comment:
Maybe merge this with the existing `delete_pyc_files!` method to avoid traversing the entire formula prefix twice to delete some very closely related file types? Or is it reasonable to expect that we would want to turn these on and off independently of each other?